pkg/dd/monitor.go

```diff
@@ -0,0 +1,51 @@
+package dd
+
+import (
+	"fmt"
+	"strconv"
+
+	aw "github.com/deanishe/awgo"
+	"gopkg.in/zorkian/go-datadog-api.v2"
+)
+
+type Monitor struct {
+	DatadogWorkflowInterface
+	client    *datadog.Client
+	cacheName string
+	wf        *aw.Workflow
+}
+
+func NewMonitor(client *datadog.Client, wf *aw.Workflow) Monitor {
+	return Monitor{
+		cacheName: monitorCacheName(),
+		client:    client,
+		wf:        wf,
+	}
+}
+
+func (d *Monitor) ListMonitors() error {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported method Monitor.ListMonitors should have comment or be unexported](https://app.codacy.com/manual/nekottyo/alfred-datadog-workflow/pullRequest?prid=5421329)